docs/checklists/medtronic600.md

```diff
@@ -0,0 +1,236 @@
+
+#### Boluses
+
+  - `[x]` normal bolus
+    - `[x]` amount of insulin delivered
+    - `[x]` amount of insulin delivery programmed (if differs from actual delivery, in case of bolus interruption, cancellation, etc.)
+  - `[x]` extended bolus
+    - `[x]` amount of insulin delivered
+    - `[x]` duration of insulin delivery
+    - `[x]` amount of insulin delivery programmed (if differs from actual delivery, in case of bolus interruption, cancellation, etc.)
+    - `[x]` duration of insulin delivery programmed (if differs from actual duration, in case of bolus interruption, cancellation, etc.)
+    - `*[ ]` extended bolus that crosses midnight is split into two records
+  - `[x]` combo/dual bolus
+    - `[x]` amount of insulin delivered - immediate (normal)
+    - `[x]` amount of insulin delivered - extended
+    - `[x]` duration of extended insulin delivery
+    - `[x]` amount of immediate insulin delivery programmed (if differs from actual delivery, in case of bolus interruption, cancellation, etc.)
+    - `[x]` amount of extended insulin delivery programmed (if differs from actual delivery, in case of bolus interruption, cancellation, etc.)
+    - `[x]` duration of extended insulin delivery programmed (if differs from actual duration, in case of bolus interruption, cancellation, etc.)
+    - `*[ ]` extended portion of combo bolus that crosses midnight is split into two records
+  - bolus cancellations/interruptions
+    - `*[ ]` represented by a separate event in the device's data log OR
+    - `*[ ]` result in modifications to a bolus event in the device's data log
+  - `[x]` link to "wizard"/calculator entry (via log entry ID or similar)
+
+No Tidepool data model yet:
+
+  - bolus cancellations/interruptions
+    - `[ ]` agent/reason for bolus cancellation
+
+Device-specific? (Add any device-specific notes/additions here.)
+
+#### CBG
+
+(See [the CGM checklist](CGMChecklist.md) instead.)
+
+#### Device Events
+
+  - alarms:
+    - `[x]` low insulin
+    - `[x]` no insulin
+        - `[ ]` needed to infer a suspend (stoppage of all insulin delivery)
+    - `[ ]` low power
+    - `[ ]` no power
+        - `[ ]` needed to infer a suspend (stoppage of all insulin delivery)
+    - `[ ]` occlusion
+        - `[ ]` needed to infer a suspend (stoppage of all insulin delivery)
+    - `[ ]` no delivery
+        - `[ ]` needed to infer a suspend (stoppage of all insulin delivery)
+    - `[ ]` auto-off
+        - `[ ]` needed to infer a suspend (stoppage of all insulin delivery)
+    - `[ ]` over limit (i.e., max bolus exceeded through override)
+    - `[ ]` other alarm types (details to be provided in `payload` object)
+  - `[x]` prime events
+    - `[x]` prime target = tubing
+    - `[x]` prime target = cannula
+    - `[ ]` prime targets not differentiated
+    - `[x]` prime volume in units of insulin
+  - `[x]` reservoir change (or reservoir rewind)
+    - `*[ ]` needed to infer a suspend (stoppage of all insulin delivery)
```

Comment:
Does the pump create a suspended basal record when there's a reservoir change, or do we need to create a suspended basal event?